specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,84 @@
+# Usage Unit
+
+Usage Unit refers to a unit of measurement for the purchase or consumption of resources or services. The Usage Unit for a particular line item may differ from the published Pricing Unit when providers use different units, unit increments, or dimensions for calculating cost.  This unit is commonly used when auditing or reconciling consumption or billing data.
+
+The UsageUnit column MUST be present and MUST NOT be null or empty. UsageUnit is of type String. UsageUnit should be expressed as a single unit of measure adhering to the following format options:
+
+* &lt;plural-units&gt; - "GB", "Seconds"
+* &lt;singular-unit&gt; &lt;plural-time-units&gt; - "GB Hours", "MB Days"
+* &lt;plural-units&gt;/&lt;singular-time-unit&gt; - "GB/Hour", "PB/Day"
+
+If the UsageUnit value is a composite value made from combinations of one or more units, each component MUST also align with the set of allowed values.
```

Comment:
What's the plan when the unit of measurement is not time, data or count? E.g. what if it's a distance for a particular type of service or something different? How do we ensure we're not saying it MUST be one of these 3 we've specified? If we want the spec to apply to cloud, saas and other things, it seems we can only recommend?

Reply:
See the latest update - I modified the working to be a bit more explicit on dimensions.